### PR TITLE
nostr (https://github.com/nostr-protocol/nostr)

### PR DESCRIPTION
**Pull request by temporaryaccountdontsend**
The simplest open protocol that is able to create a censorship-resistant global "social" network once and for all

### CODE REVIEW

File: pages/04.applications/99.wishlist/apps_wishlist.md [hunk 1]
```diff
@@ -196,6 +196,9 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [Nautilus](https://myactivity.stream/) | Turn your website into an ActivityPub profile  | [Upstream](https://github.com/aaronpk/Nautilus) |  |
 | Netrunner |  | [Upstream](https://github.com/mtgred/netrunner) |  |
 | [Netbird](https://netbird.io/) | Create an overlay peer-to-peer network connecting machines regardless of their location | [Upstream](https://github.com/netbirdio/netbird) | |
+| [nostr] |  |  (https://nostr.com/) | The simplest open protocol that is able to create a censorship-resistant global "social" network once and for all.
+
+It doesn't rely on any trusted central server, hence it is resilient; it is based on cryptographic keys and signatures, so it is tamperproof; it does not rely on P2P techniques, therefore it works. | [Upstream] (https://github.com/nostr-protocol/nostr) ||
```
Comment (ericgaspar) [inline pages/04.applications/99.wishlist/apps_wishlist.md:201]: The project description needs to be more concise

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: pages/04.applications/99.wishlist/apps_wishlist.md (hunk 1 above)
Comment (temporaryaccountdontsend) [reply pages/04.applications/99.wishlist/apps_wishlist.md:201]: Ok, i fix it

Comment (temporaryaccountdontsend) [reply pages/04.applications/99.wishlist/apps_wishlist.md]: Fixed

File: pages/04.applications/99.wishlist/apps_wishlist.md [hunk 2]
```diff
@@ -196,6 +196,8 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [Nautilus](https://myactivity.stream/) | Turn your website into an ActivityPub profile  | [Upstream](https://github.com/aaronpk/Nautilus) |  |
 | Netrunner |  | [Upstream](https://github.com/mtgred/netrunner) |  |
 | [Netbird](https://netbird.io/) | Create an overlay peer-to-peer network connecting machines regardless of their location | [Upstream](https://github.com/netbirdio/netbird) | |
+| [nostr] |  |  (https://nostr.com/) | The simplest open protocol that is able to create a censorship-resistant global "social" network once and for all.
+| [Upstream] (https://github.com/nostr-protocol/nostr) ||
```
Comment (tituspijean) [inline pages/04.applications/99.wishlist/apps_wishlist.md:200]:
```suggestion
| [nostr] |  |  (https://nostr.com/) | The simplest open protocol that is able to create a censorship-resistant global "social" network once and for all. | [Upstream] (https://github.com/nostr-protocol/nostr) | |
```

Comment (temporaryaccountdontsend) [reply pages/04.applications/99.wishlist/apps_wishlist.md:200]: Yes, you're right, it doesn't sound really good, i copied it from nostr repo.

Comment (temporaryaccountdontsend) [reply pages/04.applications/99.wishlist/apps_wishlist.md]: Fixed

Comment (temporaryaccountdontsend) [reply pages/04.applications/99.wishlist/apps_wishlist.md]: This is a temporary account, i log out, bye.

Comment (tituspijean) [reply pages/04.applications/99.wishlist/apps_wishlist.md]: I was merely correcting the code (one-liner, and space for the last empty column). :)

**Pull request merged**